### PR TITLE
list page (read/delete)

### PR DESCRIPTION
**Pull request by dipanjanpanja6**
compiled the code without any errors. Please have a look at this. Thanks.
### 

- [x] read, delete functionality
- [x] list route
- [x] fix firestore emulator config

Regards
Dipanjan P.
https://www.upwork.com/freelancers/~01de40c8509cbde4ff

### CODE REVIEW

File: src/components/TabPanel.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default function TabPanel(props: TabPanelProps) {
   const { children, value, index, ...other } = props
 
   return (
-    <div role="tabpanel" hidden={value !== index} id={`simple-tabpanel-${index}`} aria-labelledby={`simple-tab-${index}`} {...other}>
+    <div role="tabpanel" hidden={value !== index} id={`simple-tabpanel-${index}`} aria-labelledby={`simple-tab-${index}`} {...other} style={{ width: "100%" }}>
```
Comment (evergreen-trading-systems) [inline src/components/TabPanel.tsx:13]: Doesn't meet acceptance criteria

File: src/screens/ListPage.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from "react"
+import firebase from "firebase/app"
+import { Container, Button, List, ListItem, ListItemText, ListItemSecondaryAction } from "@material-ui/core"
+
+function ListPage() {
+  const listRef = firebase.firestore().collection("lists")
+  const [state, setState] = useState<Array<Object> | null>([])
+  useEffect(() => {
+    fetchItem()
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (evergreen-trading-systems) [inline src/screens/ListPage.tsx:10]: This a hack and can't be accepted.

**Review by evergreen-trading-systems — CHANGES_REQUESTED** [review]
This code is not correct, please reread the user story.

File: src/components/TabPanel.tsx (hunk 1 above)
Comment (dipanjanpanja6) [reply src/components/TabPanel.tsx:13]: ok, can i use makeStyles() hook of material-UI?

File: src/screens/ListPage.tsx (hunk 2 above)
Comment (dipanjanpanja6) [reply src/screens/ListPage.tsx:10]: ok, i got it.

**Pull request merged**